README.md

```diff
@@ -264,12 +265,11 @@ val reads = ac.loadAlignments(
 )
 
 // Generate, count and sort 21-mers
-val kmers =
-  reads
-    .flatMap(_.getSequence.sliding(21).map(k => (k, 1L)))
-    .reduceByKey(_ + _)
-    .map(_.swap)
-    .sortByKey(ascending = false)
+val kmers = reads.
+              flatMap(_.getSequence.sliding(21).map(k => (k, 1L))).
```

Comment:
I believe this could be left out of the commit -- I don't see any code changes here, and the style doesn't match the Scala coding style used in ADAM.